pom.xml

```diff
@@ -414,7 +411,7 @@
       <dependency>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-server</artifactId>
-        <version>${jetty-server.version}</version>
+        <version>${jetty.version}</version>
```

Comment:
Same comment here. This was done 3 days ago. Are you sure this needs to be deleted?

https://github.com/pentaho/maven-parent-poms/blame/master/pom.xml#L417

Reply:
You are right, let me fix that